src/routes/structures/[slug]/antennes/new-branch-modal.svelte

```diff
@@ -0,0 +1,78 @@
+<script lang="ts">
+  import Modal from "$lib/components/hoc/modal.svelte";
+
+  export let isOpen = false;
+</script>
+
+{#if isOpen}
+  <Modal
+    bind:isOpen
+    on:close
+    overflow
+    title="Créer une ou plusieurs antennes"
+    width="medium"
+  >
+    <h2>Qu’est-ce qu’une antenne ?</h2>
+
+    <p class="text-f16">
+      Une antenne est une <strong>nouvelle structure</strong> sur Dora
+      <strong>rattachée à votre structure principale</strong>. Une antenne a
+      ainsi la possibilité d’afficher <strong>propres offres</strong> et d’être
```

Comment:
```suggestion
      ainsi la possibilité d’afficher <strong>ses propres offres</strong> et d’être
```
Attention à la relecture ; je ne relis pas le reste, mais ça vaut sans doute le coup que tu le fasses ;-)